core-skills/web-application-development-with-react/README.md

```diff
@@ -92,29 +92,42 @@ During the assessment, the reviewer will go through your application with you as
     - A string
     - The `children` prop
 
-#### Understanding
+#### Understanding - Mark scheme
 
 The candidate can describe:
 
 - What the `key` prop is used for.
 
 ### Owl
 
-**Coming soon - Notes for the future**
-
-Owl is intended to demonstrate the ability to write tests for your application, and to integrate with external services in
-order to make a more feature rich application.
-
-Will also test componentDidMount calling APIs
-
-Forms for posting
-
-- Architecture
-  - The candidate can explain the following;
-    - Communicating with APIs
-    - Division of business logic, domain, rendering and state
-    - How to test component rendering
-    - How to test business logic
-
-**Coming Soon - Owl is intended to demonstrate ability to put understanding of advanced architecture
-principles and testing strategy into practice**
+#### Getting wise with React - Integrating with remote services
+
+For this level you are expected to demonstrate the ability to test both your React 
+components and any business logic used for communicating with remote services. It will
+also cover demonstrating best practice around the separation of business logic and UI
+logic.
+
+During the assessment, your reviewer will go through your application asking for
+demonstrations of the critera.
+
+#### Application - Mark scheme
+
+The candidate should create a React application which satisfies the following.
+
+- Has a component which displays data to the user which has been fetched from an API
+  - Candidate should showcase their component to assessor.
+  - Candidate should walk assessor through tests which give maintainers confidence
+    that the component displays data from an API.
+- Has a form component which takes information from the user and submits said information
+  to an API.
+  - Candidate should showcase their form to assessor.
+  - Candidate should demonstrate that the API has recieved information successfully
```

Comment:
Can this be via a test? Does the API have to exist?

Reply:
In Dans case he wasn't using an API which had any POST endpoints, it was purely GET.  He achieved this by:

A) Having an automated test.
B) Using the network tools to show that a POST had been made.

Reply:
Reading back my comments don't seem very clear - just seeking to clarify if an automated test mocking an API with a POST endpoint suffices to prove an API receives information.

Reply:
I will make this criterion more specific:

- Candidate should open the network tools and show that on form submission, a request was made by the browser to the API endpoint with a JSON encoded response body containing the form data.

Reply:
Yeah showing the json request would probably be a good thing